data_prep/scotland/get_reference_layers.sh

```diff
@@ -44,6 +44,28 @@ function cbd {
   wget https://nptscot.blob.core.windows.net/pmtiles/cbd_layer_2024-12-01.pmtiles -O $OUT/cbd.pmtiles
 }
 
+function recreation {
+  get_scotland_osm
+
+  # This list of tags is somewhat arbitary.
+  # I went down the amenity,leisure,tourism tags and picked the most popular ones that arguably fall under recreation.
+  # Note that we're currently only considering node geometries. This will exclude items which are defined on an area
+  # or linear geometry.
+  osmium tags-filter tmp/scotland-latest.osm.pbf \
```

Comment:
@congying-hu, here are the OpenStreetMap categories for recreation POIs for both the layer and land use density metric. The POI layer at https://a-b-street.github.io/ltn/cnt.html should have a new dot showing these. If it looks like there are any missing, we can update this list.